dask/dataframe/core.py

```diff
@@ -344,8 +344,11 @@ def __array_wrap__(self, array, context=None):
     def __array_ufunc__(self, numpy_ufunc, method, *inputs, **kwargs):
         out = kwargs.get('out', ())
         for x in inputs + out:
-            if not isinstance(x, (Number, Scalar, _Frame, Array,
-                                  pd.DataFrame, pd.Series, pd.Index)):
+            # Want to check for 0-dimensional arrays
```

Comment:
Nitpick: it'd be nice if the comment explained why we're doing this, instead of what we're doing.

Reply:
That's a fair point. I can update it to something like: 
```python
# ufuncs work with 0-dimensional NumPy ndarrays
# so we don't want to raise NotImplemented
```